java/daikon/VarInfoName.java

```diff
@@ -1394,8 +1394,8 @@ protected String java_family_name(OutputFormat format, VarInfo v) {
         String term_name = null;
         if (format == OutputFormat.JML) {
           term_name = term.jml_name(v);
-        } else if (format == OutputFormat.JAVA) {
-          term_name = term.java_name(v);
+          // } else if (format == OutputFormat.JAVA) {
```

Comment:
What is the reason for this change?

Reply:
If you look up to line 1392 you'll see that the only way to get to the code in question is if format != OutputFormat.JAVA.  This is the new issue errorprone found.

Reply:
I see.  Thanks.  In that case, I would just omit it.  Leaving it commented, without an explanation, is confusing.  Thanks.

Reply:
Better, the deletion could be done in a different pull request, since it isn't _really_ related to Java 17.

Reply:
If we move this change to a different pull request then the current pull request will fail the error-prone test.  I will delete the two lines and leave the change in for now.